examples/python/ibmq/using_qiskit_terra_level_2.py

```diff
@@ -20,20 +20,11 @@
 # Import the Qiskit modules
 from qiskit import IBMQ, BasicAer
 from qiskit.circuit import QuantumCircuit
-from qiskit.circuit.library.standard_gates import SwapGate
-from qiskit.compiler import assemble
+from qiskit.compiler import transpile, assemble
```

Comment:
From the docstring above, it looks like the goal of this example is to demonstrate building a custom `PassManager`. I believe the case for using `transpile`/`assemble` is covered in `using_qiskit_terra_level_1.py`.